iree/vm/test/emitc/CMakeLists.txt

```diff
@@ -19,29 +19,29 @@ iree_cc_test(
     iree::testing::gtest
     iree::testing::gtest_main
     iree::vm
-    ::arithmetic_ops
```

Comment:
given that the common case is depending on the module static library with the implementation it'd be nice to invert this so that the non-suffixed target was the library and then a `_hdrs` one just had the header without implementation - there aren't too many cases where one would want to use the header but not have the implementation linked in and it's nice to have it called out that "hey this isn't actually a static library you're depending on, just some headers"

Reply:
Fine with this :) I have adapted this accordingly.